DigitalLearningSolutions.Web/Views/MyAccount/EditDetails.cshtml

```diff
@@ -1,95 +1,169 @@
+              default-option="Select a @customField.CustomPrompt.ToLower()"
+              select-list-options="@customField.Options"/>
+          } else {
+            <vc:text-input
+              asp-for="@("Answer" + customField.CustomFieldId)"
+              label="@(customField.CustomPrompt + (customField.Mandatory ? "" : " (optional)"))"
+              populate-with-current-value="true"
+              type="text"
+              spell-check="false"
+              autocomplete=""
+              hint-text=""
+              css-class="nhsuk-u-width-one-half"/>
+          }
+        </div>
+      </div>
+    }
+  }
+
+  <div class="nhsuk-grid-row">
+    <div class="nhsuk-grid-column-one-half">
       @{ var hintText = @"To change your profile picture, select a new image and click the Preview button to preview it.
                           To remove your profile picture click the remove button.
-                          Changes will not be made until the Save button below is clicked."; }
-      <input type="hidden" asp-for="ProfileImage"/>
-      <vc:file-input asp-for="ProfileImageFile" label="Profile picture (optional)" hint-text="@hintText" css-class="nhsuk-u-width-full"/>
-    </div>
-    <div class="nhsuk-grid-column-one-half">
-      <div class="profile-picture__image-box">
-        @if (Model.ProfileImage != null) {
-          <img class="profile-picture__image" src="data:image;base64,@Convert.ToBase64String(Model.ProfileImage)" alt="Profile Picture"/>
-        } else {
-          <img class="profile-picture__image" src="@Url.Content("~/images/avatar.png")" alt="Default Profile Picture"/>
-        }
-      </div>
-    </div>
-  </div>
-
-  <div class="nhsuk-grid-row divider">
-    <div class="nhsuk-grid-column-one-half">
-      <button name="action" class="nhsuk-button" value="previewImage">Preview</button>
-      <button name="action" class="nhsuk-button nhsuk-button--secondary" value="removeImage">Remove</button>
-    </div>
-  </div>
-
-  <div class="nhsuk-grid-row">
-    <div class="nhsuk-grid-column-full">
-      <vc:text-input asp-for="Password" label="Confirm password to save changes:" populate-with-current-value="false" type="password" spell-check="false" autocomplete="" hint-text="" css-class="nhsuk-u-width-one-half"/>
-
-      <button name="action" class="nhsuk-button" value="save">Save</button>
-    </div>
-  </div>
-</form>
-<div class="nhsuk-grid-row">
-  <div class="nhsuk-grid-column-full">
-    <vc:back-link asp-controller="MyAccount" asp-action="Index" link-text="Go back" />
-  </div>
-</div>
+                          Changes will not be made until the Save button below is clicked."; }
+      <input type="hidden" asp-for="ProfileImage"/>
+      <vc:file-input asp-for="ProfileImageFile" label="Profile picture (optional)" hint-text="@hintText" css-class="nhsuk-u-width-full"/>
+    </div>
+    <div class="nhsuk-grid-column-one-half">
+      <div class="profile-picture__image-box">

```

Comment:
My IDE complains that profile-picture__image-box does not exist - do we need this div?

Reply:
Also removed it from the Index view that I copied it from